src/pages/IstioConfigList/IstioConfigListComponent.tsx

```diff
@@ -287,40 +287,37 @@ class IstioConfigListComponent extends React.Component<IstioConfigListComponentP
     }
 
     let ruleListComponent;
-    if (this.state.loading) {
-      ruleListComponent = <PfSpinner />;
-    } else {
-      ruleListComponent = (
-        <div>
-          <NamespaceFilter
-            initialFilters={[istioTypeFilter, istioNameFilter]}
-            onFilterChange={this.filterChange}
-            onError={this.handleError}
-          >
-            <Sort>
-              <Sort.TypeSelector
-                sortTypes={sortFields}
-                currentSortType={this.state.currentSortField}
-                onSortTypeSelected={this.updateSortField}
-              />
-              <Sort.DirectionSelector
-                isNumeric={false}
-                isAscending={this.state.isSortAscending}
-                onClick={this.updateSortDirection}
-              />
-            </Sort>
-          </NamespaceFilter>
-          <ListView>{istioList}</ListView>
-          <Paginator
-            viewType="list"
-            pagination={this.state.pagination}
-            itemCount={this.state.istioItems.length}
-            onPageSet={this.pageSet}
-            onPerPageSelect={this.pageSelect}
-          />
-        </div>
-      );
-    }
+    ruleListComponent = (
+      <div>
+        <PfSpinnerContainer />
```

Comment:
looking at the code this new code replaces, the old code only shows the NamespaceFilter <div> when not loading. This new code seems to always have the NamespaceFilter that follows the spinner. Is this what we want, or do we want to hide the rest of the content if isLoading is true? Does this namespace filter code break somehow or do we not want it to show if we are currently loading?

Reply:
Ok, good point. Changing.